src/collections/programs/lfx-2024/lfx-2024.mdx

```diff
@@ -59,9 +59,9 @@ Part-time terms will start on the same schedule and last six versus three months
 
 <h3>About our Community</h3>
  
-Layer5 is all about <Link to="/community">its community</Link> of contributors. We have designed an onboarding program customized to <Link to="/community/newcomers">meet newcomers</Link> where they’re at and developed an onboarding buddy program, <Link to="/community/meshmates">MeshMates</Link> with individuals dedicated to assisting contributors. Layer5 and Meshery have been around for a year and half and have a healthy, growing community of 800+ contributors. The layer5.io website itself is open source and created by 400+ of our contributors.
+Layer5 is all about <Link to="/community">its community</Link> of contributors. We have designed an onboarding program customized to <Link to="/community/newcomers">meet newcomers</Link> where they’re at and developed an onboarding buddy program, <Link to="/community/meshmates">MeshMates</Link> with individuals dedicated to assisting contributors. Layer5 and Meshery have been around for a year and a half and have a thriving community of 1900+ contributors. The layer5.io website itself is open source and created by more than 1000 of our contributors.
```

Comment:
The line `around for a year and half `needs to be updated with more specific. // @vishalvivekm can you?